scene/gui/control.cpp

```diff
@@ -1475,10 +1475,18 @@ void Control::set_rotation(real_t p_radians) {
 	_notify_transform();
 }
 
+void Control::set_rotation_degrees(real_t p_degrees) {
+	set_rotation(Math::deg_to_rad(p_degrees));
+}
+
 real_t Control::get_rotation() const {
 	return data.rotation;
 }
 
+real_t Control::get_rotation_degrees() const {
+	return Math::rad_to_deg(data.rotation);
```

Comment:
```suggestion
	return Math::rad_to_deg(get_rotation());
```
For consistency mostly.